### ISSUE TITLE
Incorrectly claiming I am using a server workspace

### ISSUE THREAD

**Issue by cruising** [Microsoft/vsts-vscode#315]
Hi Guys,
I've been trying to set this up and am getting this error when I open my solution folder:

> (team) It appears you are using a Server workspace. Currently, TFVC support is limited to Local workspaces.

It also contains a VS2017 .sln file as I only use VS Code for frontend stuff - not sure if it makes any difference.
Checking the workspace from VS2017 indicates it is a local workspace.
This is from the settings:
{
    "tfvc.location": "C:\\Program Files (x86)\\Microsoft Visual Studio\\2017\\Professional\\Common7\\IDE\\CommonExtensions\\Microsoft\\TeamFoundation\\Team Explorer\\tf.exe",
    "team.showWelcomeMessage": false
}
We are using Visual Studio Team Services.

Thanks,
Casper

**Comment by GuilhermeGHN** [comment]
Hi Casper,

I went through the same inconvenience when trying to configure, I even deleted and created the workspace to see if it worked and nothing ...

After many attempts can be made to configure via command prompt, with the following command:

C:\Program Files (x86)\Microsoft Visual Studio\2017\Professional\Common7\IDE\CommonExtensions\Microsoft\TeamFoundation\Team Explorer\tf.exe vc workspace "Computer;Owner" /location:local

After executing the command will appear the same configuration dialog box when executed via VS 2017, in this screen it is enough to confirm, after that the VS Code started working with the extension TFVC.

I hope this information helps you.

PS. Sorry for my English, I used Translator.
